### PR TITLE
stake2earn Pledge

### PR DESCRIPTION
**Pull request by AlexDCryptoC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # Agoric Validator Pledge
 
-_context: see Jan 28 [Validator Delegation Program](https://agoric.com/blog/announcements/agoric-validator-program/) announcement._
+I, stake2earn hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```
Comment (jeetraut) [inline README.md:3]:
```suggestion
[stake2earn Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1dm5l7udmdc66lza5dvhtmqw0m6u640qc0c2ymg)
I, stake2earn hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.: 
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Review by AlexDCryptoC — COMMENTED** [review]
Sure

**Comment by jeetraut** [comment]
Hi. Thanks for the contribution, but we need your README.md file to be in a subfolder. Take a look at https://github.com/Agoric/validator-profiles/pull/2 for an example where the README.md file is BlockShark/README.md.

We could try to resolve the conflicts in this branch / PR, but that would be kind of a hassle, so... can we ask you to just make another PR?

**Pull request closed without merging**